### PR TITLE
	Anime.js library insertion (javascript animation library)

### PR DESCRIPTION
**Pull request by al-amiir**
- It was complicated to make webpack output css file with multi js files (easy to output one css file with one js file), sass compile used to do that insted of wepback , then both outputs generated
- In life-system introduction use (`npm run webpack-scss:production`) insted of (`npm run build:css`)
- Because webpack folder's name changed from (`webpac-demo`) to (`webpack`) a lot of #new file and #deleted appear in git
- Chart is now simple, no multi chart buttons

### CODE REVIEW

**Review by mohamed-abdul-fattah — COMMENTED** [review]
Thanks for your effort. Please, check my comments.

**Comment by mohamed-abdul-fattah** [comment]
`webpack-demo` directory is still there. Please, consider removing it.

**Comment by mohamed-abdul-fattah** [comment]
Kindly, inspect the home page and locate these scripts
```html
<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.4/Chart.bundle.min.js" 
        integrity="sha512-SuxO9djzjML6b9w9/I07IWnLnQhgyYVSpHZx0JV97kGBfTIsUYlWflyuW4ypnvhBrslz1yJ3R+S14fdCWmSmSA==" 
        crossorigin="anonymous">
</script>
<script defer src="http://localhost:3000/dist/home.js"></script>
```
They are not in the `head` tag neither in `body`

File: public/layouts/header.php [hunk 1]
```diff
@@ -9,7 +9,8 @@
           rel="stylesheet"
           integrity="sha384-WskhaSGFgHYWDcbwN70/dfYBj47jz9qbsMId/iRN3ewGhXQFZCSftd1LZCfmhktB"
           crossorigin="anonymous">
-<script defer src="<?php echo url('dist/scss.js') ?>"></script>
+    <link rel="stylesheet" href="<?php echo url('dist/css/main.css') ?>">
+    <script src="https://kit.fontawesome.com/fe6cb3607a.js" crossorigin="anonymous"></script>
```
Comment (mohamed-abdul-fattah) [inline public/layouts/header.php:13]: Font awesome styles are already imported in [`main.scss`](https://github.com/mohamed-abdul-fattah/life-system/blob/fcfdcde39f3c9081fc4308ca208362aa0b140f2b/webpack/src/scss/main.scss#L4). Why do you need this font awesome script? What does it do while font awesome is a font style?.

**Review by mohamed-abdul-fattah — COMMENTED** [review]
Thanks @El-amiir . Please, check my comments.

File: webpack/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "scss:watch": "sass ./src/scss/main.scss ../public/dist/css/main.css -w",
+    "wepback:watch": "webpack -w",
+    "webpack-scss:watch": "run-p scss:watch wepback:watch ",
+    "scss": "sass ./src/scss/main.scss ../public/dist/css/main.css",
+    "production": "webpack",
+    "webpack-scss:production": "run-p scss production "
+  },
+  "keywords": [],
+  "author": "Elamir <engineeralamiir@gmail.com>",
+  "license": "ISC",
```
[the remark this answers is not in the document]
Comment (El-amiir) [reply webpack/package.json:17]: Done

File: webpack/package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    "npm-run-all": "^4.1.5",
+    "sass": "^1.32.12",
+    "sass-loader": "^11.0.1",
+    "style-loader": "^2.0.0",
+    "webpack": "^5.36.2",
+    "webpack-cli": "^4.6.0"
+  },
+  "dependencies": {
+    "animejs": "^3.2.1",
+    "node-sass": "^5.0.0"
```
[the remark this answers is not in the document]
Comment (El-amiir) [reply webpack/package.json:29]: Done

File: webpack/src/js/chart.js [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+        });
+      },
+    });
+  }
+};
+
+btnCardDisplay.addEventListener("click", function (e) {
+  e.preventDefault();
+  if (cardChart.style.height === "" || cardChart.style.height === "0rem") {
+    console.log(window.innerWidth);
```
[the remark this answers is not in the document]
Comment (El-amiir) [reply webpack/src/js/chart.js:168]: Done

File: webpack/src/js/chart.js [hunk 5]
```diff
@@ -0,0 +1,183 @@
+// import { reset, cardAnimation } from "./globalFunctions";
```
[the remark this answers is not in the document]
Comment (El-amiir) [reply webpack/src/js/chart.js:1]: Done

File: webpack/package.json [hunk 6]
```diff
@@ -1,25 +1,31 @@
 {
-  "name": "webpack-demo",
+  "name": "my-monitor",
   "version": "1.0.0",
   "description": "",
   "private": true,
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
```
[the remark this answers is not in the document]
Comment (El-amiir) [reply webpack/package.json:7]: Done

File: webpack/src/js/chart.js [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+};
+
+let html;
+let legendBox;
+let legends = document.querySelector(".legend");
+let cardChart = document.querySelector(".card__chart");
+let chartContainer = document.querySelector(".card__chart--container");
+let btnCardDisplay = document.querySelector(".card__btns--display");
+
+// Chart animation
```
[the remark this answers is not in the document]
Comment (El-amiir) [reply webpack/src/js/chart.js]: Done

**Review by mohamed-abdul-fattah — APPROVED** [review]

File: public/layouts/header.php (hunk 1 above)
Comment (El-amiir) [reply public/layouts/header.php:13]: Same behavior, but I did it through Font awesome website, not bootstrap website

File: webpack/src/scss/components/_sign-form.scss [hunk 8]
```diff
@@ -0,0 +1,62 @@
+.form {
```
[the remark this answers is not in the document]
Comment (El-amiir) [reply webpack/src/scss/components/_sign-form.scss]: Done

**Review by mohamed-abdul-fattah — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**